deploy/helm/values.yaml

```diff
@@ -1,6 +1,11 @@
 # Default values for security-profiles-operator.
 replicaCount: 3
 
+
+spoimage:
```

Comment:
let's use camel-case for this. e.g. we should use `spoImage` instead to be consistent with other keys.